pkgs/tools/security/passage/default.nix

```
@@ -22,15 +23,21 @@ stdenv.mkDerivation {
     sha256 = "1val8wl9kzlxj4i1rrh2iiyf97w9akffvr0idvbkdb09hfzz4lz8";
   };
 
+  patches = lib.optional stdenv.isDarwin [ ./darwin-getopt-path.patch ];
```

Try doing something like

```suggestion
  patches = [
    (substituteAll {
      src = ./darwin-getopt-path.patch;
      GETOPT = "${getopt}/bin/getopt";
    })
  ];
```

---

Ah right, we can actually patch this on all platforms if we want as it's just unused on others.